bin/appliance_console

```diff
@@ -411,6 +412,18 @@ Static Network Configuration
         # Get the region again because it may have changed
         region = ManageIQ::ApplianceConsole::DatabaseConfiguration.region
 
+        queue_options = {
+          "Use an external queue" => "external_queue",
```

Comment:
Let's use the term "messaging system", I'm thinking, even calling this external_messaging

Reply:
Really good explanation of message queues from Microsoft [here](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/dn589781(v=pandp.10)).  Agree that the critical term is messaging, not queueing.

Reply:
:+1: